build/ci/set_environment.sh

```diff
@@ -31,9 +31,13 @@ else
 fi
 git_tag="v${version}"
 
+git_repository_lower=$(echo "${GIT_REPOSITORY}" | tr '[:upper:]' '[:lower:]')
```

Comment:
```suggestion
git_repository_lower="${GIT_REPOSITORY,,}"
```

Also, I think this should be called `GITHUB_REPOSITORY` instead, as it's a GitHub specific concept — git just has remotes (which are full URLs, or at least relative file paths).

Reply:
Isn't `${VAR,,}` a Bash 4 thing?

Reply:
Actions use Bash 4.4.